imports/server/assets.ts

```diff
@@ -55,7 +54,7 @@ const hashes = [
   });
 });
 
-MeteorPromise.awaitAll(hashes);
+await Promise.all(hashes);
```

Comment:
Are...are we allowed to just have awaits at global scope?  https://github.com/meteor/meteor/issues/12287 suggests that we can't yet.

Reply:
I also found that PR, but when I tested locally this seemed to work. I suspect it's possible that one will block the application from starting up and one won't, in which case potentially if you load the app very quickly after the server starts you might see problematic behavior (although maybe we should use `Meteor.startup` to get that behavior explicitly).

Reply:
Did some very basic testing with a `while true; do curl ...` loop, and according to that test there was no race at startup. I'm not willing to say definitively that there's no race, but if there is it's narrow enough that I don't feel obliged to track it down.